tests/StacktraceTest.php

```diff
@@ -297,6 +297,25 @@ public function testGetFrameArgumentsDoesNotModifyCapturedArgs(): void
         $this->assertEquals($result['param2']['key'], 'xxxxx');
     }
 
+    public function testPreserveXdebugFrameArgumentNames(): void
+    {
+        $frame = [
+            'file' => __DIR__ . '/resources/a.php',
+            'line' => 9,
+            'args' => [
+                'foo' => 'bar',
+                'alice' => 'bob',
+            ],
+            'function' => 'a_test',
+        ];
+
+        $stacktrace = new Stacktrace(new Options(['max_value_length' => 5]), $this->serializer, $this->representationSerializer);
```

Comment:
Any reason you set the `max_value_length` option?

Reply:
Fixed